<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CFCconfigWindow</class>
 <widget class="QWidget" name="CFCconfigWindow">
  <property name="windowModality">
   <enum>Qt::WindowModality::NonModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>421</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Config</string>
  </property>
  <property name="windowIcon">
   <iconset resource="ClevoFanControl.qrc">
    <normaloff>:/ClevoFanControl/ClevoFanControl.ico</normaloff>:/ClevoFanControl/ClevoFanControl.ico</iconset>
  </property>
  <property name="toolTip">
   <string>Clevo Fan Control</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0">
    <layout class="QGridLayout" name="gridLayout" rowstretch="10,0,0,0" columnstretch="1,1,0">
     <item row="3" column="0">
      <widget class="QPushButton" name="pushButton">
       <property name="text">
        <string>OK</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1" colspan="2">
      <widget class="QPushButton" name="pushButton_2">
       <property name="text">
        <string>Apply</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QGroupBox" name="groupBox_2">
       <property name="title">
        <string>CPU Fan</string>
       </property>
       <layout class="QGridLayout" name="gridLayout_5">
        <item row="0" column="0">
         <layout class="QGridLayout" name="gridLayout_4">
          <item row="0" column="0">
           <widget class="QRadioButton" name="radioButton">
            <property name="text">
             <string>Temp-Speed</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QTableWidget" name="tableWidget">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <attribute name="horizontalHeaderVisible">
             <bool>true</bool>
            </attribute>
            <attribute name="horizontalHeaderCascadingSectionResizes">
             <bool>false</bool>
            </attribute>
            <attribute name="horizontalHeaderMinimumSectionSize">
             <number>100</number>
            </attribute>
            <attribute name="horizontalHeaderDefaultSectionSize">
             <number>100</number>
            </attribute>
            <attribute name="horizontalHeaderShowSortIndicator" stdset="0">
             <bool>false</bool>
            </attribute>
            <attribute name="verticalHeaderVisible">
             <bool>false</bool>
            </attribute>
            <column>
             <property name="text">
              <string>Temp</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>Speed</string>
             </property>
            </column>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QRadioButton" name="radioButton_2">
            <property name="text">
             <string>Max Temp</string>
            </property>
           </widget>
          </item>
          <item row="3" column="0">
           <layout class="QGridLayout" name="gridLayout_3">
            <item row="0" column="0">
             <widget class="QLabel" name="label_3">
              <property name="text">
               <string>Speed Up Temp</string>
              </property>
             </widget>
            </item>
            <item row="2" column="1">
             <widget class="QLineEdit" name="lineEdit_4"/>
            </item>
            <item row="0" column="1">
             <widget class="QLineEdit" name="lineEdit"/>
            </item>
            <item row="1" column="0">
             <widget class="QLabel" name="label_4">
              <property name="text">
               <string>Slow Down Temp</string>
              </property>
             </widget>
            </item>
            <item row="3" column="1">
             <widget class="QLineEdit" name="lineEdit_5"/>
            </item>
            <item row="3" column="0">
             <widget class="QLabel" name="label_7">
              <property name="text">
               <string>Speed Step</string>
              </property>
             </widget>
            </item>
            <item row="1" column="1">
             <widget class="QLineEdit" name="lineEdit_2"/>
            </item>
            <item row="2" column="0">
             <widget class="QLabel" name="label_6">
              <property name="text">
               <string>Min Speed</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
     <item row="1" column="1" colspan="2">
      <widget class="QGroupBox" name="groupBox_3">
       <property name="title">
        <string>GPU Fan</string>
       </property>
       <layout class="QGridLayout" name="gridLayout_18">
        <item row="0" column="0">
         <layout class="QGridLayout" name="gridLayout_13">
          <item row="0" column="0">
           <widget class="QRadioButton" name="radioButton_11">
            <property name="text">
             <string>Temp-Speed</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QTableWidget" name="tableWidget_6">
            <attribute name="horizontalHeaderVisible">
             <bool>true</bool>
            </attribute>
            <attribute name="horizontalHeaderCascadingSectionResizes">
             <bool>false</bool>
            </attribute>
            <attribute name="horizontalHeaderMinimumSectionSize">
             <number>100</number>
            </attribute>
            <attribute name="horizontalHeaderDefaultSectionSize">
             <number>100</number>
            </attribute>
            <attribute name="horizontalHeaderShowSortIndicator" stdset="0">
             <bool>false</bool>
            </attribute>
            <attribute name="verticalHeaderVisible">
             <bool>false</bool>
            </attribute>
            <column>
             <property name="text">
              <string>Temp</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>Speed</string>
             </property>
            </column>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QRadioButton" name="radioButton_12">
            <property name="text">
             <string>Max Temp</string>
            </property>
           </widget>
          </item>
          <item row="3" column="0">
           <layout class="QGridLayout" name="gridLayout_17">
            <item row="2" column="1">
             <widget class="QLineEdit" name="lineEdit_29"/>
            </item>
            <item row="3" column="1">
             <widget class="QLineEdit" name="lineEdit_0"/>
            </item>
            <item row="1" column="0">
             <widget class="QLabel" name="label_29">
              <property name="text">
               <string>Slow Down Temp</string>
              </property>
             </widget>
            </item>
            <item row="3" column="0">
             <widget class="QLabel" name="label_32">
              <property name="text">
               <string>Speed Step</string>
              </property>
             </widget>
            </item>
            <item row="2" column="0">
             <widget class="QLabel" name="label_31">
              <property name="text">
               <string>Min Speed</string>
              </property>
             </widget>
            </item>
            <item row="0" column="0">
             <widget class="QLabel" name="label_28">
              <property name="text">
               <string>Speed Up Temp</string>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="QLineEdit" name="lineEdit_26"/>
            </item>
            <item row="1" column="1">
             <widget class="QLineEdit" name="lineEdit_27"/>
            </item>
           </layout>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
     <item row="2" column="0" colspan="3">
      <widget class="QGroupBox" name="groupBox">
       <property name="title">
        <string>Others</string>
       </property>
       <layout class="QGridLayout" name="gridLayout_23">
        <item row="0" column="0" colspan="2">
         <layout class="QGridLayout" name="gridLayout_22" columnstretch="0,0,0,0,0,0,0">
          <item row="1" column="0">
           <widget class="QLabel" name="label_2">
            <property name="text">
             <string>Static Speed</string>
            </property>
           </widget>
          </item>
          <item row="2" column="2">
           <widget class="QLineEdit" name="lineEdit_36"/>
          </item>
          <item row="3" column="3">
           <widget class="QLabel" name="label_9">
            <property name="text">
             <string>GPU Fan Interval(ms)</string>
            </property>
           </widget>
          </item>
          <item row="2" column="3">
           <widget class="QLabel" name="label_42">
            <property name="text">
             <string>GPU Fan</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QLabel" name="label_39">
            <property name="text">
             <string>CPU Fan</string>
            </property>
           </widget>
          </item>
          <item row="2" column="4">
           <widget class="QLineEdit" name="lineEdit_38"/>
          </item>
          <item row="1" column="2">
           <widget class="QLineEdit" name="lineEdit_35"/>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="label_38">
            <property name="text">
             <string>Speed Limit</string>
            </property>
           </widget>
          </item>
          <item row="1" column="5" colspan="2">
           <widget class="QCheckBox" name="checkBox">
            <property name="text">
             <string>Enabled</string>
            </property>
           </widget>
          </item>
          <item row="1" column="3">
           <widget class="QLabel" name="label_41">
            <property name="text">
             <string>GPU Fan</string>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QLabel" name="label_40">
            <property name="text">
             <string>CPU Fan</string>
            </property>
           </widget>
          </item>
          <item row="2" column="5" colspan="2">
           <widget class="QCheckBox" name="checkBox_2">
            <property name="text">
             <string>Enabled</string>
            </property>
           </widget>
          </item>
          <item row="1" column="4">
           <widget class="QLineEdit" name="lineEdit_37"/>
          </item>
          <item row="3" column="2">
           <widget class="QLineEdit" name="lineEdit_6"/>
          </item>
          <item row="3" column="0" colspan="2">
           <widget class="QLabel" name="label_8">
            <property name="text">
             <string>CPU Fan Interval(ms)</string>
            </property>
           </widget>
          </item>
          <item row="3" column="4">
           <widget class="QLineEdit" name="lineEdit_7"/>
          </item>
          <item row="4" column="0" colspan="2">
           <widget class="QCheckBox" name="checkBox_3">
            <property name="text">
             <string>Use Clevo Auto</string>
            </property>
           </widget>
          </item>
          <item row="4" column="2" colspan="2">
           <widget class="QCheckBox" name="checkBox_4">
            <property name="text">
             <string>Max Speed</string>
            </property>
           </widget>
          </item>
          <item row="4" column="4" colspan="3">
           <widget class="QCheckBox" name="checkBox_5">
            <property name="text">
             <string>Monitor GPU</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QComboBox" name="comboBox"/>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Profile</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignmentFlag::AlignRight|Qt::AlignmentFlag::AlignTrailing|Qt::AlignmentFlag::AlignVCenter</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="ClevoFanControl.qrc"/>
 </resources>
 <connections/>
</ui>
